llama-index-core/llama_index/core/workflow/session.py

```diff
@@ -52,6 +54,9 @@ def send_event(self, message: Event, step: Optional[str] = None) -> None:
 
         self._broker_log.append(message)
 
+    def write_stream_event(self, ev: Optional[Event]) -> None:
```

Comment:
A bit of a nit, but just wondering if others think that `write_stream_event` may confuse some to think that this method produces a "special" kind of event?

A possible alternative could be something like `publish_event_to_stream()` or `write_event_to_stream`.

And perhaps instead of `self._streamed_events` it can be `self._event_stream`.

Reply:
Changed to `write_event_to_stream` and `self._streaming_queue`